sbws/lib/v3bwfile.py

```diff
@@ -827,17 +848,25 @@ def bw_torflow_scale(bw_lines, desc_bw_obs_type=TORFLOW_OBS_MEAN,
             # See https://trac.torproject.org/projects/tor/ticket/8494
             # just applying the formula above:
             desc_bw = min(desc_bw_obs, l.desc_bw_bur, l.desc_bw_avg)
+            if l.cons_is_unmeasured:
+                min_desc_cons_bw = desc_bw
```

Comment:
When the relay is unmeasured, this variable isn't the minimum of the descriptor and consensus bandwidths.
You could call it "unscaled_bw" instead?

Reply:
hmm, there're many more unscaled bandwidth in that file, but can not come up with a better name

Reply:
min_bw?
min_known_bw?